example/android/build.gradle

```diff
@@ -27,5 +27,6 @@ allprojects {
             // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
             url "$rootDir/../node_modules/react-native/android"
         }
+        google()
```

Comment:
Lines 23-25 already contain this maven repository. Those lines can be replaced by these.

Reply:
Updated to match a fresh React Native project 👍